### ISSUE TITLE
check if from in admin UI doesn't have negative value

### ISSUE THREAD

**Issue by Jimmi08** [e107inc/e107#4914]
## Motivation
I noticed a lot of errors in admin logs of this type:

`SELECT SQL_CALC_FOUND_ROWS e107_hpslovnik.* FROM e107_hpslovnik ORDER BY id DESC LIMIT -10, 50 `

It is caused by the negative "from" parameter in url  `?from=-10`

After searching I found that there is only an integer test for from value:
In raw mode:

`$rawData['limitFrom'] = $forceFrom === false ? (int) $request->getQuery('from', 0) : (int) $forceFrom;`

Normal way:
`$from = $forceFrom === false ? (int) $request->getQuery('from', 0) : (int) $forceFrom;`

In this case, I use the AdminUI handler in raw mode, but the behavior is the same.

## Proposed Solution

Add check - minimal value for $from should be zero in any case. 

Thanks

**Comment by CaMer0n** [comment]
@Jimmi08 Do you find these errors in the admin logs for any core database tables?

**Comment by Jimmi08** [comment]
if I add it to url, yes, 
![image](https://user-images.githubusercontent.com/5429548/206871877-f4e29f75-738f-4592-b8b0-0be88d153a51.png)

**Comment by CaMer0n** [comment]
If you add this on the next line, does it help?

    $from = abs($from);

**Comment by Jimmi08** [comment]
Yes, but it was in 2 places. Although, on second thought, maybe if it is negative then the value should be zero, which could be a better idea. 
 

![image](https://user-images.githubusercontent.com/5429548/207298989-d04c9905-547a-49e2-ad83-97558332ef02.png)
